docs/installation/installing-with-npm.md

```diff
@@ -1,303 +1,196 @@
-Here's an example of setting up [`webpack.config.js`](examples/webpack/webpack.config.js) in your project
-
-## Importing assets
-
-In order to import GOV.UK Frontend images and fonts to your project, you should configure your application to reference or copy the relevant GOV.UK Frontend assets.
-
-Follow either [Recommended solution](#recommended-solution) or [Alternative solution](#alternative-solution).
-
-### Recommended solution
-
-Make `/node_modules/govuk-frontend/govuk/assets` available to your project by routing
-requests for your assets folder there.
-
-For example, if your project uses [express.js](https://expressjs.com/), below is
-a code sample you could add to your configuration:
-
-```JS
-app.use('/assets', express.static(path.join(__dirname, '/node_modules/govuk-frontend/govuk/assets')))
+```javascript
+import { initAll } from 'govuk-frontend'
+initAll()
 ```
-### Alternative solution
-
-Manually copy the images and fonts from `/node_modules/govuk-frontend/govuk/assets` into a public facing directory in your project. Ideally copying the files to your project should be an automated task or part of your build pipeline to ensure that the GOV.UK Frontend assets stay up-to-date.
-
-The default paths used for assets are `assets/images` and `assets/fonts`. **If your asset folders follow this structure, you will not need to complete the following steps.**
-
-To use different asset paths, also complete the below step(s).
-
-1. Set `$govuk-assets-path`, `$govuk-images-path` and `$govuk-fonts-path` in your project Sass file to point to the relevant directories in your project (this will override the defaults set in `/node_modules/govuk-frontend/govuk/settings/_assets.scss`). Make sure you do this in Sass before importing `govuk-frontend` into your project - see [Importing styles](#importing-styles).
-
-  Example 1:
-
-  ``` SCSS
-  // Include images from /application/assets/images and fonts from /application/assets/fonts
-  $govuk-assets-path: '/application/assets';
-
-  @import "govuk-frontend/govuk/all";
-  ```
-
-  Example 2:
 
-  ``` SCSS
-  // Include images from /images/govuk-frontend and fonts from /fonts
-  $govuk-images-path: "/images/govuk-frontend/";
-  $govuk-fonts-path: "/fonts/";
+If you're using a bundler that uses CommonJS like [Browserify](http://browserify.org/), you should use `require`:
 
-  @import "govuk-frontend/govuk/all";
-  ```
-
-2. Optional: You can also override the helpers used to generate the asset urls, for example if you are using sass-rails' asset-pipeline functionality. You can do this by setting `$govuk-image-url-function` to the name of the function(s) you wish to use. See `src/govuk/settings/_assets.scss` for more information and examples.
+```javascript
+const GOVUKFrontend = require('govuk-frontend')
+GOVUKFrontend.initAll()
+```
 
-## Include CSS and JavaScript
+### 5. Check it works
 
-Add the CSS and JavaScript code to your HTML template:
+Create an HTML file with the following code:
```

Comment:
Should this be 'a HTML file'?

Reply:
It can look odd but "an HTML" is correct. It's because 'an' is used when the next word begins with a vowel *sound* ('*a*itch'). It's the same reason we say "a user" (starts with a 'y' consonant sound), not "an user".